webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -58,14 +58,15 @@ const CardTemplate = ({
 
   return (
     <CardContainer {...gridWidthProps} {...overrideGridProps}>
-      <Card isExpanded={isExpanded} {...cardProps}>
+      <Card isExpanded={isExpanded} ouiaId="template-card" {...cardProps}>
```

Comment:
Can this be `card-template`? later we will add a card named template and it might get the same ouiaId accidentally